src/main/java/xyz/aadev/generitech/common/tileentities/machines/TileEntityPulverizer.java

```diff
@@ -241,12 +241,12 @@ public void burnTime(int i) {
 
     @Override
     public void update() {
-
-
         //checking for the machine type
         if (machineTier == null)
             machineTier = MachineTier.byMeta(getBlockMetadata());
 
+        if (machineTier == MachineTier.TIER_0 && container.getInputRate()!=0)container.setInputRate(0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00122)